### PR TITLE
10213: document batch step

### PR DESCRIPTION
**Pull request by mkalish**
This PR adds documentation for the batch step.

Test Steps:
n/a

## Changes
- Fills out the batch.md file

### Testing
- [ ] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- #10213

### CODE REVIEW

File: prime-router/docs/universal-pipeline-overview/batch.md [hunk 1]
```diff
@@ -1,4 +1,141 @@
...
 
-This step will batch all the messages that have been received and send them all in one message to the receiver. This message can contain messages from multiple senders.
-- How to configure batch time per receiver
+This step will batch all the reports that have been translated within a configured window and send them all in one report to the receiver. 
+This message can contain messages from multiple senders for a single topic.
+
+
+## Configuration
+
+This step supports different configuration options on a **per receiver** basis:
```
[the remark this answers is not in the document]
Comment (arnejduranovic) [reply prime-router/docs/universal-pipeline-overview/batch.md:9]: Also, maybe adding a screen shot or steps on how to configure this would be helpful?

**Review by arnejduranovic — APPROVED** [review]
Amazing!!

File: prime-router/docs/universal-pipeline-overview/batch.md [hunk 2]
```diff
@@ -1,4 +1,141 @@
-# Batch
+# Batch Step
 
-This step will batch all the messages that have been received and send them all in one message to the receiver. This message can contain messages from multiple senders.
-- How to configure batch time per receiver
+This step will batch all the reports that have been translated within a configured window and send them all in one report to the receiver. 
+This message can contain messages from multiple senders for a single topic.
```
[the remark this answers is not in the document]
Comment (mkalish) [reply prime-router/docs/universal-pipeline-overview/batch.md:4]: Agreed, I'll switch to report and reserve message for things that are added to the queue

File: prime-router/docs/universal-pipeline-overview/batch.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mkalish) [reply prime-router/docs/universal-pipeline-overview/batch.md:9]: Yep I'll add some sample yaml

**Review by JFU-NAVA-PBC — APPROVED** [review]
Thanks for the details added

**Pull request merged**